Editor/CompileCesiumForUnityNative.cs

```diff
@@ -358,6 +397,9 @@ internal static void BuildNativeLibrary(LibraryToBuild library)
                     args.AddRange(library.ExtraBuildArgs);
                     startInfo.Arguments = string.Join(' ', args);
                     RunAndLog(startInfo, log, logFilename);
+                 
+                    if(library.Platform == BuildTarget.iOS)
+                        AssetDatabase.Refresh();
```

Comment:
Why is this needed on iOS specifically?

Reply:
We discussed offline that this avoids the need to create placeholders for all the .a files. We might be able to do the same on all the other platforms, too, and save some trouble. It's fine for now, though.